components/camel-stomp/src/main/java/org/apache/camel/component/stomp/StompComponent.java

```diff
@@ -55,6 +67,7 @@ protected Endpoint createEndpoint(String uri, String remaining, Map<String, Obje
         if (config.getSslContextParameters() == null) {
             config.setSslContextParameters(retrieveGlobalSslContextParameters());
         }
+        endpoint.setHeaderFilterStrategy(getHeaderFilterStrategy());
```

Comment:
Ok, I'll fix it later today. i followed like this because it was as Willem suggested so is it something wrong with this [line](https://github.com/apache/camel/blob/master/components/camel-jms/src/main/java/org/apache/camel/component/jms/JmsComponent.java#L1256)?